tests/protocol/test_wire.py

```diff
@@ -316,68 +316,6 @@ def test_get_in_vm_artifacts_profile_default(self, *args):
-        uri1.uri = 'ext_uri'
-        uris = DataContractList(ExtHandlerVersionUri)
-        uris.append(uri1)
-        host_uri = 'host_uri'
-        mock_host = HostPluginProtocol(host_uri,
-                                       'container_id',
-                                       'role_config')
-        client = WireProtocol(WIRESERVER_URL).client
-
-        with patch.object(WireClient, "fetch", return_value=None) as patch_fetch:
-            with patch.object(WireClient, "get_host_plugin", return_value=mock_host):
-                with patch.object(HostPluginProtocol, "get_artifact_request", return_value=[host_uri, {}]):
-                    HostPluginProtocol.set_default_channel(False)
-                    self.assertRaises(ExtensionDownloadError, client.fetch_manifest, uris)
-                    self.assertEqual(patch_fetch.call_count, 2)
-                    self.assertEqual(patch_fetch.call_args_list[0][0][0], uri1.uri)
-                    self.assertEqual(patch_fetch.call_args_list[1][0][0], host_uri)
-
-    # This test checks if the manifest_uri variable is set in the host object of WireClient
-    # This variable is used when we make /extensionArtifact API calls to the HostGA
-    def test_fetch_manifest_ensure_manifest_uri_is_set(self, *args):
-        uri1 = ExtHandlerVersionUri()
-        uri1.uri = 'ext_uri'
-        uris = DataContractList(ExtHandlerVersionUri)
-        uris.append(uri1)
-        host_uri = 'host_uri'
-        mock_host = HostPluginProtocol(host_uri, 'container_id', 'role_config')
-        client = WireProtocol(WIRESERVER_URL).client
-        manifest_return = "manifest.xml"
-
-        with patch.object(WireClient, "get_host_plugin", return_value=mock_host):
-            mock_host.get_artifact_request = MagicMock(return_value=[host_uri, {}])
-
-            # First test tried to download directly from blob and asserts manifest_uri is set
-            with patch.object(WireClient, "fetch", return_value=manifest_return) as patch_fetch:
-                fetch_manifest_mock = client.fetch_manifest(uris)
-                self.assertEqual(fetch_manifest_mock, manifest_return)
-                self.assertEqual(patch_fetch.call_count, 1)
-                self.assertEqual(mock_host.manifest_uri, uri1.uri)
-
-            # Second test tries to download from the HostGA (by failing the direct download)
-            # and asserts manifest_uri is set
-            with patch.object(WireClient, "fetch") as patch_fetch:
-                patch_fetch.side_effect = [None, manifest_return]
-                fetch_manifest_mock = client.fetch_manifest(uris)
-                self.assertEqual(fetch_manifest_mock, manifest_return)
-                self.assertEqual(patch_fetch.call_count, 2)
-                self.assertEqual(mock_host.manifest_uri, uri1.uri)
-                self.assertTrue(HostPluginProtocol.is_default_channel())
-
-    def test_get_in_vm_artifacts_profile_host_ga_plugin(self, *args):
```

Comment:
Already covered in `test_get_artifacts_profile_should_use_host_channel_when_direct_channel_fails`